### ISSUE TITLE
How to use UTC mode only?

### ISSUE THREAD

**Issue by r3faat1** [Hacker0x01/react-datepicker#2208]
We want the date displayed in UTC timezone. This is what gets saved on our Database anyway.

```
var d = new Date();
d -> Date Tue Jun 30 2020 14:01:25 GMT-0700 (Mountain Standard Time) // 2PM
d.toUTCString() -> "Tue, 30 Jun 2020 21:01:25 GMT" // 9PM
```

React-Datepicker should be render the time as 9PM. However, it's rendering it as 2PM due to automatically using AZ timezone.

I've looked at documentation but don't seem to see an option to ignore Timezone and just use UTC rendering.

**Comment by bipindubey-technoark** [comment]
I am also facing same issue with react-datepicker.
i have selected 12:00 Pm in datepicker and i also console log it .... it shows Thu Jul 09 2020 12:00:00 GMT+0530 (India Standard Time) which is right time.
but when i click on save then it sends wrong date (2020-07-09T06:30:00.454Z).
It should send (2020-07-09T012:00:00.454Z)----Expected Behaviour
i dont know what is the issue and how to fixx this

**Comment by tickoosunny** [comment]
This is a valid issue , has this been fixed ??

**Comment by AndrejGajdos** [comment]
+1

**Comment by 0xlarry** [comment]
+2

**Comment by jamesbhatta** [comment]
+3

**Comment by bapjiws** [comment]
Has this been addressed? I've just installed 7.4.0, and it's still silently converting the dates when submitting and then rendering back.
